tests/js/server/aql/aql-functions-string.js

```diff
@@ -1812,8 +1598,8 @@ function ahuacatlStringFunctionsTestSuite () {
 // //////////////////////////////////////////////////////////////////////////////
 
   testConcatList2: function () {
-    var expected = [ '["the","Quick","",null,"Brown",null,"アボカド","Jumps","名称について"]' ];
-    var actual = getQueryResults(`FOR r IN [ 1 ] return CONCAT([ 'the', 'Quick', '', null, 'Brown', null, 'アボカド', 'Jumps', '名称について' ], null)`);
+    var expected = [ '["the","Quick","",null,"Brown",null,"アボカド","Jumps","名称について"]false' ];
```

Comment:
Why the `false` here?
```suggestion
    var expected = [ '["the","Quick","",null,"Brown",null,"アボカド","Jumps","名称について"]' ];
```